src/txn/txn_rollback_to_stable.c

```diff
@@ -437,9 +426,24 @@ __txn_rollback_to_stable_check(WT_SESSION_IMPL *session)
 static int
 __txn_rollback_to_stable_btree_apply(WT_SESSION_IMPL *session)
 {
+    WT_CONFIG ckptconf;
+    WT_CONFIG_ITEM cval, durableval, key;
     WT_CURSOR *cursor;
     WT_DECL_RET;
-    const char *uri;
+    WT_TXN_GLOBAL *txn_global;
+    wt_timestamp_t newest_durable_ts;
+    wt_timestamp_t rollback_timestamp;
```

Comment:
These two should be declared on the same line.

Reply:
Fixed.